list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
**Observations**: The code does not handle cases where the database query might fail or return an error.

**Recommendations**: Implement error handling to manage query failures gracefully.

**Example**:
```php
$query = "SELECT * FROM books";
$result = mysqli_query($connection, $query);
if (!$result) {
    echo "Error: " . mysqli_error($connection);
    exit;
}
```